### PR TITLE
Tempus tscs pid patch

### PR DESCRIPTION
**Pull request by sconde**
@trilinos/tempus

### CODE REVIEW

**Comment by sconde** [comment]
@jmgate can we get this patch merged?

File: packages/tempus/src/Tempus_TimeStepControlStrategyPID.hpp [hunk 1]
```diff
@@ -139,6 +138,7 @@ class TimeStepControlStrategyPID
      }
      tscsPL_->validateParametersAndSetDefaults(*this->getValidParameters());
 
+     strategy_name_ = tscsPL_->get<std::string>("Name");
```
Comment (seamill) [inline packages/tempus/src/Tempus_TimeStepControlStrategyPID.hpp:141]: @sconde Why don't we need the "template" keyword here? I don't see a difference between this and what we had.

**Review by jmgate — APPROVED** [review]

**Review by seamill — APPROVED** [review]

**Comment by seamill** [comment]
@jmgate It looks like an unrelated panzer error. Can we hotfix this fix into the EMPIRE fork for now?

**Comment by jmgate** [comment]
Hotfixed into EM-Plasma/Trilinos.  Why are we seeing a Panzer failure here, though?

**Comment by seamill** [comment]
Given how today is going, I'm just going to say its just another glitch in the matrix. We can try testing again in a few hours and if it fails again we will look into it. Our change should not affect anything on the Panzer side, only the downstream codes like Drekar/EMPIRE.

**Pull request merged**